.circleci/config.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - run:
           name: Start Gradle daemon
           command: ./gradlew
-
```

Comment:
Nit: I'd leave such empty lines, for the sake of delimiting logical sections of the pipeline